pantheon-bundle/frontend/src/app/assemblyDisplay.tsx

```diff
@@ -271,10 +281,33 @@ class AssemblyDisplay extends Component<any, IAssemblyDisplayState> {
     }
 
     private onPublishEvent = () => {
-        this.getVersionUUID(this.props.location.pathname)
-        setTimeout(()=> {
-            this.getPortalUrl(this.props.location.pathname.substring(PathPrefixes.MODULE_PATH_PREFIX.length), this.state.variant)
-        }, 500)
+        // try to get the url from api only if the component has been published
+        if (this.state.releaseUpdateDate.trim() !== ""
+        && this.state.releaseUpdateDate.length >= 15 ?
+            new Intl.DateTimeFormat("en-GB", { year: "numeric", month: "long", day: "numeric" }).format(new Date(this.state.releaseUpdateDate)) : "--") {
```

Comment:
Observational Comment:
Do we need to use date in this way ?

Reply:
Thanks for the review. Once CCS-3860 is done, the whole publish event itself would change.

Reply:
Anyways updated to simplify the logic.